parsl/dataflow/memoization.py

```diff
@@ -1,7 +1,10 @@
 import hashlib
 from functools import singledispatch
 import logging
-from parsl.executors.serialize.serialize import serialize_object
+# from parsl.executors.serialize.serialize import serialize_object
+from parsl.serialize import ParslSerializer
```

Comment:
from a mostly style point of view, can these globals be defined in parsl.serialize and imported?

`from parsl.serialize import serialize_object`  reads much better to me than this block of code